sky/clouds/kubernetes.py

```diff
@@ -64,6 +64,9 @@ class Kubernetes(clouds.Cloud):
                                                              'tiers are not '
                                                              'supported in '
                                                              'Kubernetes.',
+        clouds.CloudImplementationFeatures.HOST_CONTROLLERS: 'Kubernetes can '
```

Comment:
cc'ing @cblmemo to add clouds that should be black list for hosting the controllers.